charts/k8s-monitoring/templates/agent_config/_profiles_ebpf.river.txt

```diff
@@ -0,0 +1,63 @@
+{{ define "agent.config.profilesEbpf" }}
+// Profiles
+{{- if .Values.profiles.ebpf.enabled }}
+discovery.kubernetes "ebpf_pods" {
+  selectors {
+    role = "pod"
+    field = "spec.nodeName=" + env("HOSTNAME")
+  }
+{{- if .Values.profiles.ebpf.namespaces }}
+  namespaces {
+    names = {{ .Values.profiles.ebpf.namespaces | toJson }}
+  }
+{{- end }}
+  role = "pod"
+}
+
+discovery.relabel "ebpf_pods" {
+  targets = discovery.kubernetes.ebpf_pods.targets
+  rule {
+    action = "drop"
+    regex = "Succeeded|Failed|Completed"
```

Comment:
Do you need `Pending` here as well? For consistency with `discovery.relabel.pprof_pods`

Reply:
@simonswine ?

Reply:
No, having `Pending` ones included is fine for ebpf.

Although I noticed that according to https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase Completed is not a valid state